### ISSUE TITLE
2.1.1 Is missing a template file

### ISSUE THREAD

**Issue by warmans** [flowup/api-client-generator#25]
It looks like `ngx-module-export.mustache` does not exist in the released version?

```
/usr/bin/ngx-swag-client -> /usr/lib/node_modules/@flowup/ngx-swagger-client-generator/dist/main.js
+ @flowup/ngx-swagger-client-generator@2.1.1
added 26 packages in 6.849s
Error encored during generating { Error: ENOENT: no such file or directory, open '/usr/lib/node_modules/@flowup/ngx-swagger-client-generator/dist/../templates/ngx-module-export.mustache'
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '/usr/lib/node_modules/@flowup/ngx-swagger-client-generator/dist/../templates/ngx-module-export.mustache' }
```

**Comment by warmans** [comment]
I guess it should be in this list: https://github.com/flowup/api-client-generator/blob/master/package.json#L15

**Comment by vmasek** [comment]
Yes, this was fixed in 3.alpha.1, but I will create bugfix also for version 2  :)

**Comment by vmasek** [comment]
Issue should be fixed in 2.1.2

Also please notice that project name has changed and repo has migrated (also on NPM). Please use the new one https://www.npmjs.com/package/api-client-generator
